tests/core/tests/test_admin_integration.py

```diff
@@ -536,6 +537,64 @@ def has_export_permission(self, request):
         with self.assertRaises(PermissionDenied):
             m.get_export_data('0', Book.objects.none(), request=request)
 
+    def test_export_admin_action_none_formats(self):
```

Comment:
Is this mis-named?  Should it be `test_export_admin_action_one_format()` ?

Reply:
@matthewhegarty 
I see. I changed the method name.